src/main/java/net/morbz/osmonaut/osm/Entity.java

```diff
@@ -135,4 +133,9 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
 		id = in.readLong();
 		tags = (Tags)in.readObject();
 	}
+
+	@Override
```

Comment:
same here - is there any reason why `HashCodeBuilder` is needed instead of the default java objects implementation (as proposed by intelliJ)?